include/revng/PipelineC/Prototypes.h

```diff
@@ -7,6 +7,7 @@
 #include "PipelineC.h"
 
 /**
+ * OWNING POINTERS
```

Comment:
There's probably a Doxygen way to do this other than all caps. Can you look it up?

Reply:
Writing here for this issue and the block with `INVALIDATIONS`
I've pushed changes that should generate a separate page for revng's PipelineC conventions, so they are all in one place, together with this all functions that use invalidation and/or errors now have a doxygen link that, if clicked, will bring the user to the conventions page, highlighting the appropriate heading.